pom.xml

```diff
@@ -43,8 +43,8 @@
   </scm>
   <developers>
     <developer>
-      <id>gabrielasmeria</id>
-      <name>Gabriela Smeria</name>
+      <id>acotiuga</id>
+      <name>Alexandru Cotiuga</name>
```

Comment:
But what if the past developer just started an idea and created some pages without any working result? Basically this application consisted in a macro call and 3 translation keys. Everything is changed now. How should I proceed in this case? Thanks!